src/components/Footer/index.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import CONFIG from '@/lib/configs';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import { P } from '@/styles/text';
+import FooterImage from '../../../public/images/footer-image.webp';
+import Icon from '../Icon';
+import * as Styles from './styles';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  {
+    href: CONFIG.facebook,
+    type: 'facebook',
+  },
+  { href: CONFIG.xTwitter, type: 'x' },
+  {
+    href: CONFIG.linkedin,
+    type: 'linkedin',
+  },
+  { href: CONFIG.instagram, type: 'instagram' },
+];
+
+export default function Footer() {
+  return (
+    <Styles.FooterContainer>
+      <Styles.FooterDiv>
+        <Styles.FooterSection>
+          <Image
+            alt="footer image"
+            src={FooterImage.src}
+            placeholder="blur"
+            blurDataURL={FooterImage.src}
+            quality={100}
+            width={168}
+            height={66}
+          />
+          <hr style={{ width: '160px', color: 'white' }} />
+          <P $color="white">
+            2727 Camino del Rio South, Suite 320 San Diego, CA 92108
+          </P>
+          <P $color="white">©2024 by ABA Immigration Justice Project</P>
+          <Flex
+            $direction="row"
+            $gap="10px"
+            $py="40px"
+            $px="0px"
+            $align="center"
+          >
+            <Styles.FooterLink
+              $color="white"
+              href="https://calblueprint.org/"
+              style={{ fontSize: '0.938rem' }}
+            >
+              Made By Cal Blueprint
+            </Styles.FooterLink>
+            <Icon type="blueprint" />
+          </Flex>
+        </Styles.FooterSection>
+      </Styles.FooterDiv>
+      <Styles.FooterDiv>
+        <Styles.FooterSection>
+          <Styles.Header>Resources</Styles.Header>
+          <hr style={{ width: '40px', color: 'white' }} />
```

Comment:
nit: let's stay consistent and use `2.5rem` here as well

you may want to turn this `hr` into a separate styled component just to abstract away the styling

Reply:
additionally, we should add a margin-bottom of `10px` to the horizontal line, since there's some extra spacing under the line in the design:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/04cc2265-9405-49b2-a761-6bb88121d3f0)